docs/spec/src/integration/spec/5-lifecycle-phases.md

```diff
@@ -90,77 +90,95 @@ message BlobStatusReply {
   BlobVerificationInfo blob_verification_info = 3;
 }
 
-// Intermediate states: QUEUED, ENCODED
-// Terminal states: CERTIFIED, UNKNOWN, FAILED, INSUFFICIENT_SIGNATURES
+// Intermediate states: QUEUED, ENCODED, GATHERING_SIGNATURES
+// Terminal states: UNKNOWN, COMPLETE, FAILED
 enum BlobStatus {
   UNKNOWN = 0; // functionally equivalent to FAILED but for unknown unknown bugs
   QUEUED = 1; // Initial state after a DisperseBlob call returns
   ENCODED = 2; // Reed-Solomon encoded into chunks ready to be dispersed to DA Nodes
-  CERTIFIED = 3; // blob has been dispersed and attested by NA nodes
-  FAILED = 4; // permanent failure (for reasons other than insufficient signatures)
-  INSUFFICIENT_SIGNATURES = 5;
+  GATHERING_SIGNATURES = 3; // blob chunks are actively being transmitted to validators
+  COMPLETE = 4; // blob has been dispersed and attested by NA nodes
+  FAILED = 5;
 }
 ```
 
-After a successful DisperseBlob rpc call, `BlobStatus.QUEUED` is returned. To retrieve a `cert`, the `GetBlobStatus` rpc shall be polled until a terminal status is reached. If `BlobStatus.CERTIFIED` is received, the `signed_batch` and `blob_verification_info` fields of the `BlobStatusReply` will be returned and can be used to create the `cert` . Any other terminal status indicates failure, and a new blob dispersal will need to be made.
+After a successful DisperseBlob RPC call, the disperser returns `BlobStatus.QUEUED`. To retrieve a cert, the GetBlobStatus RPC should be polled until a terminal status is reached.
 
-**Failover to EthDA**
+If `BlobStatus.GATHERING_SIGNATURES` is returned, the `signed_batch` and `blob_verification_info` fields will be present in the `BlobStatusReply`. These can be used to construct an `DACert`, which may be verified immediately against the configured threshold parameters stored in the `EigenDACertVerifier` contract. If the verification passes, the certificate can be accepted early. If verification fails, polling should continue.
 
-The proxy can be configured to retry `FAILED` dispersal n times, after which it returns to the rollup a `503` HTTP status code which rollup batchers can use to failover to EthDA. See [here](https://github.com/ethereum-optimism/specs/issues/434) for more info.
+Once `BlobStatus.COMPLETE` is returned, the `signed_batch` and `blob_verification_info` fields are guaranteed to be included and should be used to construct the final `DACert`.
+
+ Any other terminal status indicates failure, and a new blob dispersal will need to be made.
+
+**Failover to Native Rollup DA**
+
+The proxy can be configured to retry `FAILED` dispersal `n` times, after which it returns to the rollup a `503` HTTP status code which rollup batchers can use to failover to EthDA or native rollup DA offerings (e.g, arbitrum anytrust). See [here](https://github.com/ethereum-optimism/specs/issues/434) for more info on the OP implementation and [here](https://hackmd.io/@epociask/SJUyIZlZkx) for Arbitrum. 
```

Comment:
prob want to retry not only on `FAILED` but also on `UNKNOWN` and even `COMPLETE` (see my  msg above) right? Not sure how to best translate that to words but think it would be more accurate

Reply:
true but thats probably out-of-scope of this PR